tests/contrib/test_dictionary_storage.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for oauth2client.contrib.dictionary_storage"""
+
+import unittest2
+
+from oauth2client import GOOGLE_TOKEN_URI
+from oauth2client.client import OAuth2Credentials
+from oauth2client.contrib.dictionary_storage import DictionaryStorage
+
+
+class DictionaryStorageTests(unittest2.TestCase):
+
+    def _generate_credentials(self, scopes=None):
```

Comment:
Private method that makes no use of `self` - always pull these out of their classes and into module-scope module-private helper functions.